pkg/ingester/ingester.go

```diff
@@ -509,12 +541,20 @@ func (i *Ingester) Describe(ch chan<- *prometheus.Desc) {
 	ch <- i.queries.Desc()
 	ch <- i.queriedSamples.Desc()
 	ch <- i.memoryChunks.Desc()
+
+	i.memSeriesTotal.Describe(ch)
```

Comment:
Are you missing a Describe on `ingestedSamplesFail`?

Reply:
Also, if there is not difference between the `ch <-` form and this form, lets prefer consistency.

Reply:
Interestingly, `cortex_ingester_ingested_samples_failures_total` still shows up in prometheus.

Reply:
```
pkg/ingester/ingester.go:595:5: cannot use i.memSeriesTotal (type *prometheus.CounterVec) as type prometheus.Metric in send:
	*prometheus.CounterVec does not implement prometheus.Metric (missing Desc method)
```